### PR TITLE
I made some changes you will absolutly like :)

### PR DESCRIPTION
**Pull request by YaronKoresh**
Hey!
these are the changes I added:
* option to choose colors (not random)
* better order and sizes of the left panel
* some little errors (eg. unnecessery '/no-wait') have been fixed
* the resize of the window works well with another "on-detect" event (just for critical things)
Hope you will like it!

### CODE REVIEW

File: forms.red [hunk 1]
```diff
@@ -1,111 +1,152 @@
...
-	}
-	History: [
-		0.1.0 "22-08-2017"	"Start of work."
-		0.3.4 "26-03-2018"	"Source editor split"
-		0.3.5 "30-04-2018"	"Dynamic code arrangement"
-		0.3.6 "30-07-2018"	"Fixed font size typo"
-		0.3.7 "27-08-2018"	"Fixed on-drop issue"
-		0.3.8 "05-10-2018"	"Fixed widgets toolbox issue"
-	]
-]
```
Comment (planetsizecpu) [inline forms.red]: Why have you removed the red info and history header? I want to preserve this info, you may add your nickname as contributor if you want.

File: forms.red [hunk 2]
```diff
@@ -1,111 +1,152 @@
...
+staticEditorDefSize: as-pair StaticEditorDefXsize EditorDefYsize
+StaticDefOrigin: as-pair 0 (FormSheetDefYorigin + FormSheetDefYsize + 5)
+DynamicDefOrigin: as-pair StaticEditorDefXsize (FormSheetDefYorigin + FormSheetDefYsize + 5)
 DynamicCode: does copy [" "]
 
-; Database default values
-DbDefOrigin: as-pair (WindowDefXsize - (ToolboxDefXsize + 10)) FormSheetDefYorigin
-DbDefXsize: FormSheetDefXsize
-DbDefYsize: ToolboxDefYsize - 120
-DbDefSize: as-pair DbDefXsize DbDefYsize
```
Comment (planetsizecpu) [inline forms.red:68]: This code was here for preserving space as the I/O Branch would permit work with DB Servers, but I really don't care :-)

File: forms.red [hunk 3]
```diff
@@ -1,111 +1,152 @@
...
+        txt "Blue:"  B: slider 256 value react [face/text: to-text B/data]
+	
+        pad 0x-65 box: base react [face/color: to-color R/data G/data B/data] return
+
+        pad 0x20 text "The new color"
+            font  [size: 14]
+            react [face/font/color: box/color]
+        button "Done!" [do bacg box/color]
+    ]
+]
```
Comment (planetsizecpu) [inline forms.red]: Good job!
Before the random color, it was a full enabled color selector func from @honyx but along the time view was enhanced and the func become faulty, so I changed it by random as a time saving option 😂

Comment (planetsizecpu) [reply forms.red]: So I like your selector!

File: forms.red (hunk 1 above)
Comment (planetsizecpu) [reply forms.red]: Please, restore the old header, add your changes to history and set you as contributor!

Comment (YaronKoresh) [reply forms.red]: I just thought its not nessery, and it take some lines...

File: forms.red (hunk 3 above)
Comment (YaronKoresh) [reply forms.red]: thanks!
I thought to add your random color selector as a background color for the main window.
What do you say?

File: forms.red (hunk 1 above)
Comment (YaronKoresh) [reply forms.red]: I have edited this but its not being updated.. I checked you in "allow edits by maintainer"

**Comment by planetsizecpu** [comment]
Good job, the header its necessary for people as me to remember, I'm a bit old man :-)  also as Red is a open language you can accesss the header in source code by system/...  You better investigate yourself ;-)

**Pull request merged**